tests/test_edgeql_ddl.py

```diff
@@ -8114,7 +8114,7 @@ async def test_edgeql_ddl_constraint_03(self):
                 """)
 
     @test.xfail('''
-        EXISTS constraint violation not raised for MULTI property.
+        Reports an schema error. Maybe that is exactly what we want?
```

Comment:
Ideally, we should support this, but raising SchemaError is a lot better than accepting it and then _not_ raising the constraint violation.

Reply:
I guess we could enforce complex constraints manually at the DML side...

Reply:
Same way we enforce `required`.